features/model/step_definitions.rb

```diff
@@ -25,7 +25,9 @@
 
 When("we run the rails command line with:") do |cmd|
   if cmd.start_with?('g aws_record:model')
-    @table_name = cmd.split(' ')[2]
+    @model_name = cmd.split(' ')[2]
+    @table_name = SecureRandom.uuid
```

Comment:
As a matter of course, I'd do:

```ruby
@table_name = "#{@model_name}_#{SecureRandom.uuid}"
```

That way, you keep the model name for readability.